querybook/server/datasources_socketio/ai_assistant.py

```diff
@@ -0,0 +1,31 @@
+from flask import request
+from flask_socketio import join_room, leave_room
+
+from const.ai_assistant import (
+    AI_ASSISTANT_NAMESPACE,
+    AI_ASSISTANT_REQUEST_EVENT,
+)
+
+from .helper import register_socket
+
+
+@register_socket("subscribe", namespace=AI_ASSISTANT_NAMESPACE)
+def on_join_room():
+    join_room(request.sid)
```

Comment:
do you need to join rooms for individual websocket requests? i thought those are automatic

Reply:
you're right